src/bidders/prebid/index.js

```diff
@@ -163,6 +158,31 @@ export class Prebid extends BaseBidder {
 		);
 	}
 
+	refreshBids(slotAlias) {
+		if (!this.refreshSlots.includes(slotAlias)) {
+			this.refreshSlots.push(slotAlias);
```

Comment:
Do we really need this array? Wouldn't it be better to have one source of truth and check context to decide whether  given ad unit can be refreshed?

Reply:
That's a good idea. Refactored.